src/gui/Unix/QMPlay2.appdata.xml.in

```diff
@@ -12,25 +12,27 @@
     </description>
     <url type="homepage">https://github.com/zaps166/QMPlay2</url>
     <url type="bugtracker">https://github.com/zaps166/QMPlay2/issues</url>
-    <categories>
-        <category>AudioVideo</category>
-        <category>Player</category>
-        <category>Audio</category>
-        <category>Video</category>
-        <category>Qt</category>
-    </categories>
-    <keywords>
-        <keyword>player</keyword>
-        <keyword>audio</keyword>
-        <keyword>video</keyword>
-    </keywords>
+    <url type="vcs-browser">https://github.com/zaps166/QMPlay2</url>
+    <url type="translate">https://github.com/zaps166/QMPlay2/tree/master/lang</url>
     <content_rating type="oars-1.1"/>
-    <releases>
-        <release version="@QMPLAY2_VERSION@"/>
-    </releases>
     <project_license>LGPL-3.0</project_license>
     <developer_name>Błażej Szczygieł</developer_name>
+    <launchable type="desktop-id">io.github.zaps166.QMPlay2.desktop</launchable>
     <screenshots>
-        <screenshot type="default">https://raw.githubusercontent.com/zaps166/GitHubCommonContents/master/Screenshots/QMPlay2.png</screenshot>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/zaps166/GitHubCommonContents/master/Screenshots/QMPlay2.png</image>
+        </screenshot>
     </screenshots>
+    <releases>
```

Comment:
I'd prefer `@QMPLAY2_VERSION@`, CMake should replace the placeholder.

Reply:
In that case, you end up with missing release 

```
E: QMPlay2.desktop:27: release-time-missing date
   The release is missing either the `date` (preferred) or the `timestamp` property.
```

Reported issue: https://github.com/ximion/appstream/issues/570

Reply:
Date can be by CMake, too, by parsing QMPlay2 version.

Reply:
Done: https://github.com/zaps166/QMPlay2/commit/9f08396b70937ba7c1fda561a7d63f6a15f2bccd